packages/@o3r/design/builders/generate-css/index.ts

```diff
@@ -101,7 +101,7 @@ export default createBuilder<GenerateCssSchematicsSchema>(async (options, contex
       await renderDesignTokens(tokens, renderDesignTokenOptions);
       return { success: true };
     } catch (err) {
-      return { success: false, error: `${err as any}` };
```

Comment:
Why do we do this change ?

Reply:
I suggested this change in a previous comment.
For me it makes more sense to use `String(err)` since we are essentially just casting the Error to string.
Using template string to do a string cast seems misleading, it also requires a `as any` which is not pretty